robot/web/web.go

```diff
@@ -354,7 +354,6 @@ func (svc *webService) refreshResources() error {
 }
 
 func (svc *webService) updateResources(resources map[resource.Name]resource.Resource) error {
```

Comment:
What's the intended behavior of this function? Is it to walk all of the "registered `ResourceAPICollection` objects and place each item of `resources` input into the right collection?

And most importantly (given the bug) -- the `resources` input represents *exactly* what should exist? And any prior resource that's no longer in this `resources` input should be deleted?

Reply:
yep, every existing resource on the robot's resource graph gets passed into this function. The function should then update the ResourceAPICollection objects with the correct resources, include deleting ones which no longer exist

Reply:
Cool, can we document that?

Reply:
added